### ISSUE TITLE
The Youtube page from which the video is played has broken UI after quitting from immersive mode and resuming the app

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2177]
**[Affected Versions]:**
- Firefox Reality 6-33100022 RC3 (1e9e4a85) build.

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- Oculus Go - Android 7.1.2
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2

**[Prerequisites]:**
- The headset is turned on and the app is opened.

**[Steps to reproduce]:**
1. Open and Play a 360 video in immersive mode
2. Click the "Oculus/Vive" button from the controller.
3. Click the "Quit/Home" button from the prompt displayed.
4. Resume the app and observe the Youtube page.

**[Expected results]:**
- The page is correctly rendered.

**[Actual results]:**
- The page's suggestions and comments sections are blank and the page is not scrollable.

**[Notes]:**
- The page UI is redisplayed correctly after it's refreshed.
- Only the previously opened video is displayed in the center of the page and in some attempts it's displayed with no controls and with a black preview.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1Wo4hqpVPRymZ6eY5bTIR93DmF3M8SyCQ)

**Comment by MortimerGoro** [comment]
This also happens without entering immersive video mode before. So it seems related to a session restore issue

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the Firefox Reality 6-33112229 RC4 (f2e90af) build. Now, the Youtube website's UI is not broken after quitting and reopening the app.
However, after reopening the app the YouTube UI is being non-responsive on smaller windows than 2x which is documented in issue #2134.

Tested on the Oculus Quest (Android 7.1.1), Oculus Go (Android 7.1.2), HTC Vive Focus Plus (Android 7.1.2) and HTC Vive Focus (Android 7.1.2) devices.
